components/automl/import_data_from_gcs/component.py

```diff
@@ -0,0 +1,39 @@
+from typing import NamedTuple
+
+
+def automl_import_data_from_gcs(
+    dataset_path: str,
+    input_uris: list,
+    retry=None, #=google.api_core.gapic_v1.method.DEFAULT,
+    timeout=None, #=google.api_core.gapic_v1.method.DEFAULT,
+    metadata: dict = None,
+) -> NamedTuple('Outputs', [('dataset_path', str)]):
+    import sys
+    import subprocess
+    subprocess.run([sys.executable, "-m", "pip", "install", "google-cloud-automl==0.4.0", "--quiet", "--no-warn-script-location"], env={"PIP_DISABLE_PIP_VERSION_CHECK": "1"}, check=True)
```

Comment:
1. Recommended  using ' as quote delimiter  to be consistent across.
2. pip3

Reply:
Done

Reply:
> 2\. pip3

It's the module name here, not the program name =)